src/tools/compiletest/src/runtest.rs

```diff
@@ -359,6 +361,17 @@ impl<'test> TestCx<'test> {
         self.check_forbid_output(&output_to_check, &proc_res);
     }
 
+    fn run_crash_test(&self) {
+        let pm = self.pass_mode();
+        let proc_res = self.compile_test(WillExecute::No, self.should_emit_metadata(pm));
+
+        // if a test does not crash, consider it an error
+        match proc_res.status.code() {
+            Some(101) => (),
+            _ => self.fatal("expected ICE"),
```

Comment:
Can you elaborate this message to tell the dev to give the test a better message and move it to `tests/ui/something`?

Specifically, I don't want people to just copy over the `122909.rs` to `tests/ui` when they go from fail -> pass.